bundles/org.openhab.binding.e3dc/src/main/resources/ESH-INF/i18n/e3dc_de.properties

```diff
@@ -0,0 +1,29 @@
+# binding
+binding.e3dc.name = E3DC Hauskraftwerk Binding
+binding.e3dc.description = Integriert die Hauskraftwerk Modelle von E3DC. Zeigt die Werte der elektrischen Produzenten und Konsumenten sowie die Einstellungen des E3DC Hauskraftwerkes. Angeschlossene Wallboxen k�nnen gesteuert werden.
+
+# thing types
+thing-type.e3dc.e3dc-device.label = E3DC Hauskraftwerk
+thing-type.e3dc.e3dc-device.description = Verbindung zum E3DC Hauskraftwerk. Nach dem Einrichten f�gen Sie Ihre gew�nschten Bl�cke zu dieser Bridge hinzu.
+thing-type.e3dc.e3dc-info.label = E3DC Info Block
+thing-type.e3dc.e3dc-info.description = Basis Informationen zum E3DC Hauskraftwerk wie Modellname, Seriennummer und Softwareversionen
+thing-type.e3dc.e3dc-power.label = E3DC Leistung Block
+thing-type.e3dc.e3dc-power.description = Elektrische Werte der angeschlossenen Produzenten (Photovoltaik, Batterie, ...) und Konsumenten (Haushaltsverbrauch, Wallbox,...) 
+thing-type.e3dc.e3dc-wallbox.label = E3DC Wallbox Einstellungen Block
+thing-type.e3dc.e3dc-wallbox.description = Zeigt die Einstellungen der verbundenen Wallbox. Schalter wie "Sonnenmodus" oder "1-phasiges Laden" k�nnen ge�ndert werden!
+thing-type.e3dc.e3dc-strings.label = E3DC String Details Block
+thing-type.e3dc.e3dc-strings.description = Detaillierte Werte der angeschlossenen Photovoltaik Strings. Zeigt die einzelnen Werte jedes angeschlossenen Strings
+thing-type.e3dc.e3dc-emergency.label = E3DC Notstrom Block
+thing-type.e3dc.e3dc-emergency.description = Status und Werte der Notstromversorgung wie Einspeisungsbeschr�nkung, Batterie Lade- und Entladesperren
+
+# thing type config description
+thing-type.config.e3dc.e3dc-device.host.label = IP Adresse oder Host Name
+thing-type.config.e3dc.e3dc-device.host.description = Netzwerkadresse oder Hostname des E3DC Ger�tes
+thing-type.config.e3dc.e3dc-device.deviceid.label = Ger�tenummer
+thing-type.config.e3dc.e3dc-device.deviceid.description = Ger�tenummer aus den Modbus Einstellungen des E3DC Ger�tes
+thing-type.config.e3dc.e3dc-device.port.label = Modbus Port
```

Comment:
```suggestion
thing-type.config.e3dc.e3dc-device.port.label = Modbus-Port
```

Reply:
Obsolete due to usage of Modbus TCP Bridge